collector/start_service/start_service_test.go

```diff
@@ -0,0 +1,64 @@
+package start_service
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/SUSE/sap_host_exporter/internal/sapcontrol"
+	"github.com/SUSE/sap_host_exporter/test/mock_sapcontrol"
+	"github.com/golang/mock/gomock"
+	"github.com/prometheus/client_golang/prometheus/testutil"
+	"github.com/stretchr/testify/assert"
```

Comment:
afaik in the `ha_cluster_exporter` we have an internal pkg for assertion. why are we importing here the `testify`?

Reply:
We use `testify/assert` there too. We don't need a custom assertion for the metrics at the moment, so the internal one is not here yet.